dropwizard-util/src/test/java/io/dropwizard/util/JavaVersionTest.java

```diff
@@ -0,0 +1,25 @@
+package io.dropwizard.util;
+
+import org.testng.annotations.Test;
```

Comment:
Dropwizard 1.3.x is using JUnit 4.12 for testing. Please change the class accordingly.

Reply:
Changed to Junit